modules/virt-accessing-vm-yaml-ssh.adoc

```diff
@@ -0,0 +1,169 @@
+metadata:
+  namespace: ssh-ns <1>
+  name: vm-ssh
+spec:
+  running: false
+  template:
+    metadata:
+      labels:
+        kubevirt.io/vm: vm-ssh
+        special: vm-ssh <2>
+    spec:
+      domain:
+        devices:
+          disks:
+          - disk:
+              bus: virtio
+            name: containerdisk
+          - disk:
+              bus: virtio
+            name: cloudinitdisk
+          interfaces:
+          - masquerade: {} <3>
+            name: testmasquerade <4>
+          rng: {}
+        machine:
+          type: ""
+        resources:
+          requests:
+            memory: 1024M
+      networks:
+      - name: testmasquerade
+        pod: {}
+      volumes:
+      - name: containerdisk
+        containerDisk:
+          image: kubevirt/fedora-cloud-container-disk-demo
+      - name: cloudinitdisk
+        cloudInitNoCloud:
+          userData: |
+            #!/bin/bash
+            echo "fedora" | passwd fedora --stdin
+...
+----
+<1> Name of the namespace created by the `oc create namespace` command.
+<2> Label used by the service to identify the virtual machine instances that are enabled for SSH traffic connections. The label can be any `key:value` pair that is added as a `label` to this YAML file and as a `selector` in the service YAML file.
+<3> The interface type is `masquerade`.
+<4> The name of this interface is `testmasquerade`.
+
+. Create the VM:
++
+[source,terminal,subs="+quotes"]
+----
+$ oc create -f __<path_for_the_VM_YAML_file>__
+----
++
+. Start the VM:
++
+[source,terminal]
+----
+$ virtctl start vm-ssh
+----
++
+. In the YAML file for the service, specify the service name, port number, and the target port.
++
+.Example `Service` definition
+[source,yaml]
+----
+...
+apiVersion: v1
+kind: Service
+metadata:
+  name: svc-ssh <1>
+  namespace: ssh-ns <2>
+spec:
+  ports:
+  - targetPort: 22 <3>
+    protocol: TCP
+    port: 27017
+  selector:
+    special: vm-ssh <4>
+  type: NodePort
+...
+----
+<1> Name of the SSH service.
+<2> Name of the namespace created by the `oc create namespace` command.
+<3> The target port number for the SSH connection.
+<4> The selector name and value must match the label specified in the YAML file for the VM.
+
+. Create the service:
++
+[source,terminal,subs="+quotes"]
+----
+$ oc create -f __<path_for_the_service_YAML_file>__
+----
+
+. Verify that the VM is running:
++
+[source,terminal]
+----
+$ oc get vmi
+----
++
+.Example output
+[source,terminal]
+----
+NAME    AGE     PHASE       IP              NODENAME
+vm-ssh 6s       Running     10.244.196.152  node01
+----
+
+. Check the service to find out which port the service acquired:
++
+[source,terminal]
+----
+$ oc get svc
+----
++
+.Example output
+[source,terminal]
+----
+NAME            TYPE       CLUSTER-IP     EXTERNAL-IP   PORT(S)           AGE
+svc-ssh     NodePort       10.106.236.208 <none>        27017:30093/TCP   22s
+----
+
+In this example, the service acquired the port number 30093.
```

Comment:
Needs a `+` before it to stay attached to the step.

Reply:
Updated and added the +.